packages/legend-application-query/src/stores/QueryEditorStore.ts

```diff
@@ -634,6 +635,11 @@ export class MappingQueryCreatorStore extends QueryEditorStore {
         query.groupId = this.groupId;
         query.artifactId = this.artifactId;
         query.versionId = this.versionId;
+        query.taggedValues = [
+          createQueryClassTaggedValue(
```

Comment:
I actually think we don't want to do this analytics for query not created from a dataspace :( let's not do that.